### PR TITLE
Add inspector reminder

### PR DESCRIPTION
**Pull request by jaredgalanis**

### CODE REVIEW

**Comment by amyrlam** [comment]
- [x] add robert to bottom authors

File: source/2019-12-06-the-ember-times-issue-127.md [hunk 1]
```diff
@@ -35,13 +36,15 @@ Check out the full interview on [the Ember Discuss forum](https://discuss.emberj
...
-<consider adding some bold to your paragraph>
+In follow up to last week's announcement about the awesome new work being done on **Ember Inspector** to support Octane 🔥, please don't forget to help test the new changes and let the Inspector team know about your experiences (positive or negative).
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+The Inspector changes in question are currently still in a [pull request](https://github.com/emberjs/ember-inspector/pull/1088), so if you're interested in taking it for a spin you will have to follow the instructions in the PR's description. Note, these instructions are not the same as installing Inspector in the usual manner for your browser.
+
+Please feel free to report any success or issues either in the PR or in the `#st-octane` channel of Discord.
+
+While the new changes relate primarly to suppport of Octane, it would be very helpful, for anyone interested, to test this particular build of the Inspector on older versions of Ember as well. So, even if you're not enjoying Octane features yet, it would still be quite useful if you can help test things out! 😊
```
Comment (ijlee2) [inline source/2019-12-06-the-ember-times-issue-127.md:47]: I don't have suggestion mode on mobile, sorry. You have a couple of misspellings in the beginning.

**Review by ijlee2 — APPROVED** [review]
Looks good to me, much thanks for writing this!

File: source/2019-12-06-the-ember-times-issue-127.md (hunk 1 above)
Comment (amyrlam) [reply source/2019-12-06-the-ember-times-issue-127.md:47]: i think you mean `success or issues` being `success of issues` but i changed it to `success stories or issues`. may make a slight copy edit upstream. getting to the netlify view

thanks to both!!

**Pull request merged**